test/functional/feature_block.py

```diff
@@ -164,9 +164,12 @@ def run_test(self):
                 self.sign_tx(badtx, attempt_spend_tx)
             badtx.rehash()
             badblock = self.update_block(blockname, [badtx])
+            reject_reason = (template.block_reject_reason or template.reject_reason)
+            if reject_reason and reject_reason.startswith("mempool-script-verify-flag-failed"):
```

Comment:
Note: reject_reason could be None here, so a check was added (not in master commit)